### ISSUE TITLE
Run Web Code Analysis feature doesn't work in VS 2015 with .NET Core 1.0.1 tooling Preview 2

### ISSUE THREAD

**Issue by brgrz** [aspnet/Tooling#980]
After installing the .NET Core 1.0.1 Preview 2 tooling the Web Essentials/Web Analyzer code analysis tools (JSHint, ESLint, etc.) are gone and supposedly replaced by "Run Web Code Analysis" feature by right clicking the files/folders in solution but when clicking it nothing happens.

Issue might be related to https://github.com/aspnet/Tooling/issues/727

**Comment by mlorbetske** [comment]
Could you try going to `Tools -> Web Code Analysis -> Reset All Settings...` to see if it fixes the issue?

cc @madskristensen

**Comment by vasadhananjay** [comment]
I'm having the same issue. Resetting the settings doesn't appear to work.

**Comment by brgrz** [comment]
Yeah, doesn't help. Also after installing the Web Analyzer VS extension I get the following dialog popup when I start VS.

---------------------------
Web Analyzer
---------------------------
The Web Analyzer extension has now been built in to Visual Studio in the .NET Core Tooling Preview 1.

Please uninstall Web Analyzer and install .NET Core Tooling Preview 1.

Do you wish to go to the download page?
---------------------------
Yes   No   
---------------------------

Neither the supposedly built-in extension nor the one downloaded separately work.

**Comment by mlorbetske** [comment]
TFS Bug#382584
